src/view/trip.js

```diff
@@ -18,7 +18,7 @@ const getgetCreateOfferItemTemplate = (offers) => {
 };
 
 const getTripTemplate = (trips) => {
```

Comment:
Согласно KISS принципу можно упростить :)

```suggestion
const getTripTemplate = ({type, destination, timeIn, timeOut, basePrice, offers}) => {
```